### PR TITLE
Adding Readme template.

### PR DESCRIPTION
**Pull request by jah2488**
I wanted to make sure we had a working and sensible readme template to start with. So here it is.

### CODE REVIEW

**Comment by street-lit** [comment]
:+1:

**Comment by travisbm** [comment]
Great work Justin! Go ahead. :+1: +1

**Comment by bookdoctor1** [comment]
;+1:

**Comment by bookdoctor1** [comment]
:+1:

**Comment by anishaas** [comment]
:+1

**Comment by anishaas** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
 
-Things you may want to cover:
 
-* Ruby version
+## Steps to get working locally
 
-* System dependencies
+  - Clone
+  - Bundle Install
+  - Run `rake db:setup`
```
Comment (jah2488) [inline README.md]: This will cause a problem if the person who cloned the app, did so before there were any migrations.

**Comment by Adam-Ross** [comment]
:+1:

**Comment by rcyep87** [comment]
:+1: yeah!

**Comment by mmdotz** [comment]
:+1:

**Pull request merged**